docs/development/dev-process/pull-requests.md

```diff
@@ -33,7 +33,7 @@ code.
 
 Add comments to your own PR to indicate where more work will be done.
 
-Be open to suggestion and remember we are stretched thing and will likely
+Be open to suggestions and remember, we are stretched thin and will likely
```

Comment:
I disagree with the addition of the comma. I suggest one of these, instead:
- "remember we are stretched thin",
- "remember that we are stretched thin",
- "remember: We are stretched thin".

Also, I wonder if the reader is actually supposed already to know what it is told to remember.